TypeInjections/Tool/Tool.fs

```diff
@@ -21,91 +48,61 @@ module Tool =
-          [<Option('r', "remote", Required = false, HelpText = "[github repo url] [new commit] [old commit] [outputPath]")>] remote: seq<string>
-          [<Option(HelpText = "Prints all messages to standard output.")>] verbose: bool
-          }
-
-    (* command line parser has group arguments into what option it is associated with
-     (local option has a list of arguments that follow it, as does git and remote).
-     If command line option has arguments, we need to organize arguments and run typeCart on them.
-     Run typeCart only once per command line call (we can't run a remote and local project on same command line call)
-     *)
-    let interpretFlags  (local: string list) (git: string list) (remote: string list) : string * string * string =
-
-        if not local.IsEmpty then
-            Utils.log "***** local project comparison"
-            local.[0], local.[1], local.[2]
-        elif not git.IsEmpty then
-            Utils.log "***** git commit"
-            // location of repo
-            let location = Directory.GetCurrentDirectory()
-
-            let commit1, commit2 = git.[0], git.[1]
-            let path = git.[2]
-            let pathOld = $"{path}/Old"
-            let pathNew = $"{path}/New"
-
-            // lib2git needs an empty folder to store the repo clones
-            for a in [ pathOld; pathNew ] do
-                if Directory.Exists(a) then
-                    Directory.Delete(a, true)
-
-            checkoutCommit commit1 location pathOld
-            checkoutCommit commit2 location pathNew
 
-            pathOld, pathNew, $"{path}/Output"
+        let location =
+            match git.url.Length with
+            // empty git url means user wants to run typeCart in current directory
+            | 0 -> Directory.GetCurrentDirectory()
+            // if there is a repo URL, use it
+            | _ -> git.url
 
-        elif not remote.IsEmpty then
-            Utils.log "***** remote github"
 
-            let path = remote.[3]
-            let pathOld = $"{path}/Old"
-            let pathNew = $"{path}/New"
-            let url = remote.[0]
-            let commit1 = remote.[1]
-            let commit2 = remote.[2]
+        let commit1, commit2 = git.oldHash, git.newHash
+        let path = git.outputFolder
+        let pathOld = $"{path}/Old"
+        let pathNew = $"{path}/New"
 
-            for a in [ pathOld; pathNew ] do
-                if Directory.Exists(a) then
+        // lib2git needs an empty folder to store the repo clones
+        for a in [ pathOld; pathNew ] do
+            if Directory.Exists(a) then
+                // if user lets us delete existing folder
+                if git.force then
                     Directory.Delete(a, true)
+                // else break program
+                else
+                    failwith $"There is already an {a}, we need empty folder to place git repo. Chose different output folder or chose --force to automatically overwrite data"
```

Comment:
typo: choose

in general: please write correct English sentences, don't leave out articles for example

Reply:
Ugh that was lazy, sorry